common.mk

```diff
@@ -73,15 +73,15 @@ linux-common: linux-defconfig
 linux-defconfig-clean-common:
 	@rm -f $(LINUX_PATH)/.config
 
-LINUX_CLEAN_COMMON_FLAGS ?= $(LINUX_COMMON_FLAGS)
+#LINUX_CLEAN_COMMON_FLAGS ?= UNCOMMENT_AND_DEFINE_AS_NECESSARY
```

Comment:
I'm not fond of having code not actually used, even though it might sound like it is convenient. Also for the Linux related flags we must propagate the ARCH here. Today we're stating that directly at the line on the individual makefiles, like here for example: https://github.com/OP-TEE/build/blob/master/qemu.mk#L91

Maybe we should create an `ARCH` variable in the individual makefiles and just include that here?